pulpcore/pulpcore/app/viewsets/task.py

```diff
@@ -16,24 +16,34 @@ class TaskFilter(filterset.FilterSet):
     worker = HyperlinkRelatedFilter()
     started_at = filters.IsoDateTimeFilter(name='started_at')
     finished_at = filters.IsoDateTimeFilter(name='finished_at')
+    parent = HyperlinkRelatedFilter()
 
     class Meta:
         model = Task
         fields = {
             'state': ['exact', 'in'],
-            'worker': ['exact'],
+            'worker': ['exact', 'in'],
             'started_at': DATETIME_FILTER_OPTIONS,
-            'finished_at': DATETIME_FILTER_OPTIONS
+            'finished_at': DATETIME_FILTER_OPTIONS,
+            'parent': ['exact']
         }
 
 
-class TaskViewSet(NamedModelViewSet,
+class TaskViewSet(MultiSerializerMixin,
+                  NamedModelViewSet,
                   mixins.RetrieveModelMixin,
                   mixins.ListModelMixin):
+    # MultiSerializerMixin needs to come first, as the code brought in by NamedModelViewSet depends
+    # on the existence of an overridden 'get_serializer_class()' which does not need
+    # the 'serializer_class' attribute specified
```

Comment:
This makes me hesitate though. Multiple inheritance can get really complex and buggy really quickly when the order matters.